src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/mapper/RequestedFlowMapper.java

```diff
@@ -127,6 +140,32 @@ public RequestedFlow toRequestedFlow(FlowRequest request) {
     @Mapping(target = "loopSwitchId", source = "loopSwitchId")
     protected abstract RequestedFlow generatedMap(FlowRequest request);
 
+    /**
+     * Convert {@link Flow} to {@link FlowRequest}.
+     */
+    @Mapping(target = "flowId", source = "flowId")
+    @Mapping(target = "encapsulationType", source = "encapsulationType")
+    @Mapping(target = "bandwidth", source = "bandwidth")
+    @Mapping(target = "ignoreBandwidth", source = "ignoreBandwidth")
+    @Mapping(target = "periodicPings", source = "periodicPings")
+    @Mapping(target = "allocateProtectedPath", source = "allocateProtectedPath")
+    @Mapping(target = "description", source = "description")
+    @Mapping(target = "maxLatency", source = "maxLatency")
+    @Mapping(target = "priority", source = "priority")
+    @Mapping(target = "pinned", source = "pinned")
+    @Mapping(target = "detectConnectedDevices", source = "detectConnectedDevices")
+    @Mapping(target = "loopSwitchId", source = "loopSwitchId")
```

Comment:
Looks like all @Mapping above are useless because fields have same names and they will be mapped automatically

Reply:
I have cleaned up this meaningless mapper instructions (but only in this class to not extend this patch even more). More cleanups will be done in extra PRs.